packages/opentelemetry-test-utils/testUtils.ts

```diff
@@ -15,8 +15,9 @@
  */
 
 import * as childProcess from 'child_process';
-import { SpanKind, Attributes, Event, Span, Status } from '@opentelemetry/api';
 import * as assert from 'assert';
+/* eslint-disable node/no-unpublished-import */
```

Comment:
Why is this an "unpublished import"

Reply:
I don't know, perhaps, I googled a bit but nothing clear... Perhaps the package is private... But not sure.